client/src/themes/theme.ts

```diff
@@ -25,6 +25,11 @@ export const theme = createMuiTheme({
       fontFamily: "'Roboto'",
       textAlign: 'center',
     },
+    h6: {
+      fontSize: 12,
+      color: '#888888',
```

Comment:
I don't think we should mix typography and palette together - there must be a more suitable place to apply this. For example you can use global `overrides` for this if you want to change the color of every `h6` variant:
```
    overrides: {
      MuiTypography: {
        h6: {
          color: "#888888"
        }
      },
```

Reply:
Great point. Done.